e2e/support/helpers/e2e-dashboard-helpers.js

```diff
@@ -175,3 +175,7 @@ export function resizeDashboardCard({ card, x, y }) {
 export function createLinkCard() {
   cy.icon("link").click();
 }
+
+export function openDashboardMenu() {
+  return dashboardHeader().findByLabelText("dashboard-menu-button");
```

Comment:
There are some usage like this

https://github.com/metabase/metabase/blob/c9134399b7fb0ced0d667a20e558f8ada926f6a7/e2e/test/scenarios/sharing/downloads/downloads.cy.spec.js#L180